spa/src/components/common/RedirectWithReload.js

```diff
@@ -0,0 +1,9 @@
+import { useHistory, withRouter } from 'react-router-dom';
+
+const RedirectWithReload = (props) => {
+  let history = useHistory();
+  history.push({ pathname: props.to, search: props.location.search });
+  window.location.reload();
+  return <></>;
```

Comment:
Returning an empty fragment is uncommon, think just `null` would be better
```suggestion
  return null;
```

Reply:
Addressed with 02f12cff0cdaa189fe4b9b93678a7b00068cf129